File: 프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.py
def solution(s):
    l = list(s)
    l.sort()
    answer = ''
    cnt = {}
    for i in l:
        cnt[i] = cnt.get(i,0)+1
    for i in cnt.keys():
        if cnt[i] ==1:
            answer = answer+i
    return answer

File: 프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.java
import java.util.TreeMap;
class Solution {
    public String solution(String s) {
        String answer = "";
        TreeMap<Character, Integer> count = new TreeMap<>();
        for (char c : s.toCharArray()) {
            if (count.containsKey(c)) {
                count.put(c, count.get(c) + 1);
            } else {
                count.put(c,1);
            }
        }
        for (char c : count.keySet()) {
            if (count.get(c) == 1) {
                answer += c;
            }
        }
        return answer;
    }
}